docs/http/http-spans.md

```diff
@@ -74,9 +74,11 @@ HTTP server span names SHOULD be `{method} {http.route}` if there is a
 If there is no (low-cardinality) `http.route` available, HTTP server span names
 SHOULD be [`{method}`](#method-placeholder).
 
-HTTP client spans have no `http.route` attribute since client-side instrumentation
-is not generally aware of the "route", and therefore HTTP client span names SHOULD be
-[`{method}`](#method-placeholder).
+HTTP client span names SHOULD be `{method} {url.template}` if there is a
+(low-cardinality) `url.template` available (see below for the exact definition of the [`{method}`](#method-placeholder) placeholder).
+
+If there is no (low-cardinality) `url.template` available, HTTP client span names
+SHOULD be [`{method}`](#method-placeholder).
```

Comment:
I think this probably needs to be under a sections marked Experimental

And the overall doc probably needs to be marked as Status: Mixed (instead of current Status: Stable)

Reply:
Maybe we can do something like this - https://github.com/open-telemetry/semantic-conventions/pull/989 ?

Reply:
Given there is already #989 PR open, I suggest to only mark the status as defined in #989 - "[Stable][DocumentStatus], Unless otherwise specified" and add an explicit note to the Http client span naming which would use url.template + we'd keep the official "stable" version of client span naming. And then reconcile this again in #989. Otherwise we'd be trying to resolve an issue which deserves a separate PR. What do you think?

Reply:
I suspect #989 won't take long to merge since it doesn't introduce any new semantic conventions (just documents existing experimental ones), so probably more likely we can merge that first and then reconcile in this PR

Reply:
@trask that PR does not touch span naming based on experimental attribute, so whatever is merged there does not affect this PR. Hence I really want to have this one (finally) merged, if possible and reconcile experimental attributes + their usage in naming, in #989.

Reply:
I took a liberty to push some suggestions to your branch @klauco 

Let me know what you both think about it.